src/frontend/komponenter/Felleskomponenter/MånedÅrInput/ÅrVelger.tsx

```diff
@@ -34,27 +33,24 @@ const Årvelger: React.FC<ÅrProps> = ({
     antallÅrFrem,
     antallÅrTilbake,
     lesevisning = false,
-    disabled = false,
     feil = false,
 }) => {
     const årOptions = lagÅrOptions(år, antallÅrFrem, antallÅrTilbake);
     return (
-        <FamilieSelect
-            lesevisningVerdi={år ? år.toString() : ''}
+        <Select
             value={år ?? ''}
             onChange={event => {
                 event.persist();
                 settÅr(event.target.value !== '' ? parseInt(event.target.value) : undefined);
             }}
-            erLesevisning={lesevisning}
-            disabled={disabled}
+            readOnly={lesevisning}
             error={feil}
-            label={'År'}
-            hideLabel
+            label={''}
+            aria-label={'År'}
```

Comment:
Dette fikset bare problemet for lesevisning når låsen er der 😅for normal visning er nå høyden feil uansett :(
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/9c039f69-40d6-4c11-8aac-9907dc301e80)